asapdiscovery-data/asapdiscovery/data/fragalysis.py

```diff
@@ -6,7 +6,7 @@
 
 BASE_URL = "https://fragalysis.diamond.ac.uk/api/download_structures/"
 # Info for the POST call
-MPRO_API_CALL = {
+API_CALL_BASE = {
     "target_name": "Mpro",
```

Comment:
@ijpulidos should we change this to be `None` or an empty string or something?

Reply:
Done in https://github.com/choderalab/asapdiscovery/pull/255/commits/ed38a0311e293c5bacf2c0dfd293448cb8f3177e